app/src/main/java/com/example/mycalculator/mvp/CalculatorViewImpl.java

```diff
@@ -0,0 +1,22 @@
+package com.example.mycalculator.mvp;
+
+import android.widget.Button;
+import android.widget.TextView;
+
+import com.example.mycalculator.MainActivity;
+import com.example.mycalculator.databinding.ActivityMainBinding;
+
+public class CalculatorViewImpl implements CalculatorContract.calculatorView {
+
+    private ActivityMainBinding binding;
+
+    public CalculatorViewImpl(MainActivity mainActivity, ActivityMainBinding binding) {
```

Comment:
you are not using the **mainActivity** param, you can delete it

Reply:
Done. Thx!!